### PR TITLE
Add feature for admin node backup during upgrade

### PR DESCRIPTION
**Pull request by vmoravec**

### CODE REVIEW

File: features/step_definitions/upgrade/admin_backup_steps.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  expect(@backup_button.disabled?).to be(false)
+end
+
+Given(/^the "([^"]*)" button is disabled$/) do |button_text|
+  button = find_button(button_text, disabled: true)
+end
+
+When(/^I click the backup button$/) do
+  @backup_button.click
+  wait_for "Waiting for 'Next' button to get enabled", max: "10 seconds", sleep: "1 second" do
```
Comment (jsuchome) [inline features/step_definitions/upgrade/admin_backup_steps.rb:29]: Where do you get those values? By experimenting?

Comment (vmoravec) [reply features/step_definitions/upgrade/admin_backup_steps.rb:29]: yes, but they are usually double of the size/time which I experience on our cloud hw, these tests should be passing also on less performant virtual clouds.

**Review by jsuchome — APPROVED** [review]

**Comment by vmoravec** [comment]
Successful build: https://ci.suse.de/view/Cloud/view/QA/job/cloud-mkphyscloud-qa-test-upgrade-ui/23/console

**Review by skazi0 — COMMENTED** [review]
lgtm

**Pull request merged**